src/opts.rs

```diff
@@ -29,7 +29,7 @@ struct RawOpt {
 pub enum Action {
     /// Start the Eww daemon.
     #[structopt(name = "daemon")]
-    Daemon,
+    Daemon(DaemonOpts),
```

Comment:
I think it'd be cleaner to just make this a record enum variant thingy directly, so like
```rs
Daemon {
  /// Custom config path
  #[structopt(short, long)
  config: Option<std::path::PathBuf>
}
```
Should otherwise be equivalent, although you'll then need to destructure in the match in `main`, of course